.github/workflows/post-release.yml

```diff
@@ -6,69 +6,76 @@ on:
-      - name: Retag lifecycle images - semver
+          echo "LIFECYCLE_VERSION=$(echo ${{ github.event.release.tag_name }} | cut -d "v" -f2)" >> $GITHUB_ENV
+          echo "LIFECYCLE_IMAGE_TAG=$(git describe --always)" >> $GITHUB_ENV
+      - name: Get cosign public key
         run: |
-          docker pull buildpacksio/lifecycle:${{ env.LIFECYCLE_IMAGE_TAG }}-windows
-          docker image tag buildpacksio/lifecycle:${{ env.LIFECYCLE_IMAGE_TAG }}-windows buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }}-windows
-          docker push buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }}-windows
-      - name: Retag lifecycle images - latest
-        if: "!contains(env.LIFECYCLE_VERSION, 'rc')"
+          echo -n "${{ secrets.COSIGN_PUBLIC_KEY }}" | base64 --decode > cosign.pub
+      - name: Verify lifecycle images
         run: |
-          docker pull buildpacksio/lifecycle:${{ env.LIFECYCLE_IMAGE_TAG }}-windows
-          docker image tag buildpacksio/lifecycle:${{ env.LIFECYCLE_IMAGE_TAG }}-windows buildpacksio/lifecycle:latest-windows
-          docker push buildpacksio/lifecycle:latest-windows
-  create-manifest-lists:
-    runs-on: ubuntu-latest
-    needs: [retag-lifecycle-images-linux, retag-lifecycle-images-windows]
-    steps:
-      - uses: azure/docker-login@v1
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-      - name: Set env
-        run: |
-          echo "LIFECYCLE_VERSION=$(echo ${{ github.event.release.tag_name }} | cut -d "v" -f2)" >> $GITHUB_ENV
-      - name: Create lifecycle image manifest lists - semver
+          LINUX_AMD64_SHA=$(cosign verify -key cosign.pub buildpacksio/lifecycle:${{ env.LIFECYCLE_IMAGE_TAG }}-linux | jq -r .critical.image.\"docker-manifest-digest\")
+          echo "LINUX_AMD64_SHA: $LINUX_AMD64_SHA"
+          echo "LINUX_AMD64_SHA=$LINUX_AMD64_SHA" >> $GITHUB_ENV
+
+          WINDOWS_AMD64_SHA=$(cosign verify -key cosign.pub buildpacksio/lifecycle:${{ env.LIFECYCLE_IMAGE_TAG }}-windows | jq -r .critical.image.\"docker-manifest-digest\")
+          echo "WINDOWS_AMD64_SHA: $WINDOWS_AMD64_SHA"
+          echo "WINDOWS_AMD64_SHA=$WINDOWS_AMD64_SHA" >> $GITHUB_ENV
+      - name: Retag lifecycle images & create manifest list - semver
         run: |
-          DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }} buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }}-linux buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }}-windows
-          DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push buildpacksio/lifecycle:${{ env.LIFECYCLE_VERSION }}
-      - name: Create lifecycle image manifest lists - latest
+          DOCKER_CLI_EXPERIMENTAL=enabled
+
+          LINUX_AMD64_SHA=$(crane tag buildpacksio/lifecycle:${{ env.LIFECYCLE_IMAGE_TAG }}-linux@${{ env.LINUX_AMD64_SHA }} ${{ env.LIFECYCLE_VERSION }}-linux 2>&1 | cut -d' ' -f5)
```

Comment:
This is a bit brittle. I'm not sure if it's totally necessary. But it's nice to see all the shas printed in the output.